yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -170,14 +170,22 @@ func prepareImportFileTasks() []*ImportFileTask {
 	}
 	kvs := strings.Split(fileTableMapping, ",")
 	for i, kv := range kvs {
-		fileName, table := strings.Split(kv, ":")[0], strings.Split(kv, ":")[1]
-		filePath := dataStore.Join(dataDir, fileName)
-		task := &ImportFileTask{
-			ID:        i,
-			FilePath:  filePath,
-			TableName: table,
+		globPattern, table := strings.Split(kv, ":")[0], strings.Split(kv, ":")[1]
+		filePaths, err := dataStore.Glob(globPattern)
```

Comment:
why are we not joining the pattern with `dataDir` path? The pattern in file-table-mapping can be without dataDir path?

Reply:
ok, thanks @priyanshi-yb